### PR TITLE
Tweaked collector priority

### PR DESCRIPTION
**Pull request by javiereguiluz**
In https://github.com/symfony/symfony/pull/16023 we're tweaking the priorities of the toolbar/profiler panels. The `265` value is the right number for Doctrine ... but maybe you don't want to merge it yet because this will change a lot the position of the Doctrine panel in the profiler/toolbar of Symfony versions older than 2.8.

### CODE REVIEW

File: Resources/config/dbal.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         </service>
 
         <service id="data_collector.doctrine" class="%doctrine.data_collector.class%" public="false">
-            <tag name="data_collector" template="@Doctrine/Collector/db.html.twig" id="db" />
+            <tag name="data_collector" template="@Doctrine/Collector/db.html.twig" id="db" priority="265" />
```
Comment (stof) [inline Resources/config/dbal.xml]: 265 ? Wouldn't this put it before all core ones in 2.7 ?

Comment (javiereguiluz) [inline Resources/config/dbal.xml]: Yes. This is how the profiler menu looks:

profiler_menu

Comment (stof) [inline Resources/config/dbal.xml]: I'm talking about 2.7

Comment (javiereguiluz) [inline Resources/config/dbal.xml]: Sorry, I didn't push the commit. It's `250` now.

**Comment by stof** [comment]
Thank you @javiereguiluz.

**Pull request merged**